index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <a href="#" id="logo">My logo</a>
+      <div class="nav">
+        <img class="union" src="/images/Union.png" alt="img" />
+      </div>
+    </header>
+
+    <main class="main-section">
+      <h2 class="primary-text">I’m Ankit kush. Glad to see you!</h2>
```

Comment:
- [ ] Kindly I recommend you to use a `h1 t`ag for The main title (e.g., "I’m Ankit kush ...") instead of `h2` tag, If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.